.vscode/settings.json

```diff
@@ -20,42 +19,27 @@
     "**/.git": true,
     ".vscode": true,
     ".bundle": true,
-    ".gitignore": false,
-    ".gitpod.yml": false,
-    ".rspec": false,
-    ".devcontainer/": false,
-    "config.ru": false,
-    "Dockerfile": false,
-    "Gemfile": false,
-    "Gemfile.lock": false,
-    "tmp/": false,
-    "install-packages": false,
-    "main.rb": false,
-    "README.md": false,
-    "app.rb": false,
-    "environment.rb": false,
-    "bin/": false,
-    "spec/": false,
-    "examples.txt": false,
-    "config/": false
   },
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true,
-  "screencastMode.onlyKeyboardShortcuts": true,
+  "screencastMode.keyboardOptions": {
```

Comment:
I'm not actually sure what this is for, could you elaborate?

Reply:
In order to show keystrokes on the screen during screen cast mode (which can be toggled on from the command palette) we need to set what is shown. The removed line was deprecated at some point (very hard to find information about where), but eventually I found the new config settings for that by searching screencast in the vscode settings, copying the JSON, and modifying it to suit our needs here (only showing keyboard shortcuts and not every keystroke):

![image](https://github.com/appdev-projects/rails-7-template/assets/12408296/d857893e-b9e7-417e-bd09-f0686b24afac)